about.html

```diff
@@ -0,0 +1,117 @@
+       
+         <nav class="nav-bar-desktop">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li class="current"><a href="about.html">About</a></li>
+                <li><a href="index.html">Program</a></li>
+                <li><a href="about.html">Contact</a></li>
+            </ul>
+        </nav>
+        <button class = "header-button-1" type="button">CC Campaign</button>
+            
+            <div id="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div id="pop-up-mobile-window"></div>
+        </div>
+    </header>
+    
+
+    <div class="showcase-mobile">
+        <div class="container-mobile">
+            <h2 class="text-center orange-text no-display-text">"Hello! Creative World"</h2>
+            <h2 class="text-center orange-text second-heading">SCHOOL OF CREATIVE WEB DESIGN 2022</h2>
+            <div class="description-mobile-2 center-align">
+                <p class="text-center white-back">Lorem ipsum dolor sit amet 
+                consectetur adipisicing elit.
+                Nobis soluta sunt ex provident placeat 
+                similique eaque animi fugiat et.
+                Voluptates numquam, id dolorem at
+                esse doloremque deserunt magni excepturi!
+                Lorem, ipsum dolor sit amet consectetur adipisicing 
+                elit. Recusandae eius dolor.
+            </p>
+            </div>
+            <div class="announcement-mobile-2 text-center">
+                <p>please contact us per Emain for any further questions about orange Intelligence</p>
+                <h4>info@orangeintelligence.com</h4>
+            </div>
+        </div>
+    </div>
+
+    <div class="box-1 text-center">
+        <h2>Learn Mordern creative <br> Web Design</h2>
+        <div class="orange-line-mobile">
+            <span class="orange-bar-mobile">
+            </span>
+        </div>
+        <p>Lorem ipsum dolor sit amet, 
+        consectetur adipisicing elit.
+        Repellat expedita illo et aliquid Cupidatat
+         veniam officia laboris elit aliquip.Ullamco dolor id fugiat
+         sunt cupidatat culpa pariatur.</p>
+    </div>
+
+    <div class="box-2">
+        
+        <img class = "company-name-body-sec" src="./images/Orange-I-logo.jpg" alt="">
+    </div>
+
+    <section class="previous-class">
+        <h3 class="text-center">Know about our previous classes</h3>
+        <div class="orange-line-mobile">
+            <span class="orange-bar-mobile">
+            </span>
+        </div>
+            <p class="text-center">Lorem ipsum dolor sit amet consectetur 
+                adipisicing elitAb maiores architecto
+            </p>
+         <div class = "blurr-img">
+             <img src="./images/graduates-1.jpg" alt="student-img"> 
+            <img src="./images/batch-2.png" alt="student-img">
+        </div>
```

Comment:
[OPTIONAL] Kindly put the centered text in those images to match the Behance design.
| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157870754-e062bbd6-83f1-4a32-a625-0f9e8361df48.png) | ![image](https://user-images.githubusercontent.com/24703171/157870655-3fd0fe44-9b22-420d-9df3-46f69a4caa2d.png) |